task.cnf

```diff
@@ -0,0 +1,78 @@
+#
+# The MySQL database server configuration file.
+#
+# One can use all long options that the program supports.
+# Run program with --help to get a list of available options and with
+# --print-defaults to see which it would actually understand and use.
+#
+# For explanations see
+# http://dev.mysql.com/doc/mysql/en/server-system-variables.html
+
+# Here is entries for some specific programs
+# The following values assume you have at least 32M ram
+
+[mysqld]
+#
+# * Basic Settings
+#
+user	    	= root
+# pid-file	= /var/run/mysqld/mysqld.pid
+# socket  	= /var/run/mysqld/mysqld.sock
+port		    = 3000
```

Comment:
The `port` is set to `3000`, which is non-standard for MySQL (default is `3306`). Ensure this is intentional and that all clients are configured to connect to this port.